### ISSUE TITLE
Tensorboard View Scalars

### ISSUE THREAD

**Issue by phouse512** [potterhsu/SVHNClassifier#4]
Hi @potterhsu, thanks so much for documenting this, it has been very helpful to walk through. I had a question about running TensorBoard - in the training code I thought I saw references to `summary = tf.summary.merge_all()` which should send the scalar/image values to tb.

When I ran TensorBoard as specified, while I saw the graph structure loaded correctly, I couldn't see any graphs of the accuracy/loss over time.  Do they only appear after quite some time? I have only been training for less than an hour so far.

**Comment by potterhsu** [comment]
There are a few points to note:

1. TensorFlow writes log files (named something like `events.out.tfevents.xxx`) while `add_summary` is executed (https://github.com/potterhsu/SVHNClassifier/blob/master/train.py#L82)

2. TensorBoard reads log files to display scalars after it was launched, and reloads itself every 120 seconds in default, the interval can be configured using `--reload_interval` flag (see `tensorboard --help`).

So, please check if there is any log file under log directory, and relaunch TensorBoard for reloading immediately.

**Comment by phouse512** [comment]
Ah yes, this makes sense.  I first ran it on a slower machine just to test it, and I didn't realize the code publishes to tensorboard every 1000 cycles.  Now that I'm running it on a beefier machine it is working as expected - thanks!
